cmd/erasure.go

```diff
@@ -231,8 +303,8 @@ func (er erasureObjects) crawlAndGetDataUsage(ctx context.Context, buckets []Buc
 		return nil
 	}
 
-	// Collect disks we can use, sorted by least inode usage.
-	disks := er.getOnlineDisksSortedByUsedInodes()
+	// Collect disks we can use, sorted by most inode usage choose at the most (erasure stripe size)/2.
```

Comment:
This will cut crawling speed in half. I see no reason not to choose any drive not healing.

There is no reason not to use all online disks in a non-healing system.

Reply:
@klauspost there is also no reason to crawl all of them - I don't know what we gain by crawling all disks. Are you saying that usage would be inconsistent? 

- If we cut the crawling speed usage can be calculated faster
- also healing trigger `256`th cycle wouldn't need to hit all drives

Reply:
@harshavardhana You don't understand what it is doing then. It will only ever crawl one disk, but several buckets are crawled concurrently on different disks.

It will first collect buckets and then assign one bucket per disk to crawl. This means that buckets are distributed among the available disks. So if you reduce the number of available disks you reduce the crawl speed.

Distributing crawl load equally to all non-healing disks means that we distribute load as much as possible while getting the best speed possible. Reducing the number of disks will reduce speed with no benefits.

Reply:
Ah I see then let me revert it.

Reply:
Changed and pushed